vulnerabilities/migrations/0007_alter_vulnerabilityreference_reference_id.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.0.2 on 2022-04-08 18:53
+
+from django.db import migrations
+from django.db import models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("vulnerabilities", "0006_alter_advisory_unique_together"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="vulnerabilityreference",
+            name="reference_id",
+            field=models.CharField(
+                blank=True,
+                help_text="An optional reference ID, such as DSA-4465-1 when available",
+                max_length=180,
```

Comment:
Let's make it an even 200:
```suggestion
                max_length=200,
```